desktop/apps/personalize/components/RoutesWithProgressBar.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { Route, Redirect } from 'react-router'
+
+import { ProgressIndicator } from '@artsy/reaction-force/dist/Components/Onboarding/ProgressIndicator'
+import CollectorIntent from '@artsy/reaction-force/dist/Components/Onboarding/Steps/CollectorIntent'
+import Artists from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Artists'
+import Genes from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Genes'
+import Budget from '@artsy/reaction-force/dist/Components/Onboarding/Steps/Budget'
+
+const STEPS = [
+  `/personalize/${CollectorIntent.slug}`,
+  `/personalize/${Artists.slug}`,
+  `/personalize/${Genes.slug}`,
+  `/personalize/${Budget.slug}`
+]
+
+export class RoutesWithProgressBar extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      finished: false,
+    }
+  }
```

Comment:
Not necessary, but this could be simplified to exclude the constructor since we're not accessing props in the state definition.

```
export class RoutesWithProgressBar extends React.Component {
   state = {
      finished: false
   }
}
```